src/main/java/com/springweb/dv_spring_web_mongo/controller/UserController.java

```diff
@@ -23,4 +25,9 @@ public void registerNewUser(@Valid @RequestBody UserCreateOrUpdateDTO userCreate
     public void updatePassword(@Valid @RequestBody UserCreateOrUpdateDTO userCreateOrUpdateDTO) {
         userService.updatePassword(userCreateOrUpdateDTO);
     }
+
+    @PostMapping("/login")
+    public ResponseEntity<Map<Object, Object>> login(@RequestBody @Valid UserCreateOrUpdateDTO userCreateOrUpdateDTO) {
```

Comment:
It is not good idea to use the `UserCreateOrUpdateDTO` here. Please add new dto object